src/store/modules/env/env.js

```diff
@@ -0,0 +1,117 @@
+import ApiWS from '../../../apis/api-ws'
+import Api from '../../../apis/api'
+
+export default {
+	namespaced: true,
+	state() {
+		return {
+			chainId: null,
+			addrPrefix: '',
+			sdkVersion: 'Stargate',
+			apiCosmos: null,
+			apiTendermint: null,
+			apiWS: null,
+			clients: {
+				api: null,
+				ws: null
+			},
+			wallet: null,
+			getTXApi: '',
+			account: '',
+			initialized: false,
+			authorized: false
+		}
+	},
+	getters: {
+		apiClient: state => state.clients.api,
+		wsClient: state => state.clients.ws,
+		apiTendermint: state => state.apiTendermint
+	},
+	mutations: {
+		config(state, config) {
+			state.apiCosmos = config.apiNode
+			state.apiTendermint = config.rpcNode
+			if (config.wsNode) {
+				state.apiWS = config.wsNode
+			}
+		},
+		connect(state, clients) {
+			state.clients.api = clients.apiClient
+			state.clients.ws = clients.wsClient
+		},
+		initializeWSComplete(state) {
+			state.initialized = true
+		}
+	},
+	actions: {
+		async init(
+			{ dispatch },
+			config = {
+				apiNode: 'http://localhost:1317',
+				rpcNode: 'http://localhost:26657',
+				wsNode: 'ws://localhost:26657/websocket',
+				chainId: '',
+				addrPrefix: '',
+				sdkVersion: 'Stargate',
+				getTXApi: 'http://localhost:26657/tx?hash=0x'
+			}
+		) {
+			if (this._actions['modules/starport/init']) {
+				dispatch('modules/starport/init', null, { root: true })
+			} else {
+				dispatch('config', config)
+			}
```

Comment:
If starport module is registered, we hand over initialization of the environment to it instead which in turn configures the store, allowing a single initialization command regardless of whether the developer is using the starport vuex module or not